library/sysrc

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
```

Comment:
This file should be in `files/sysrc` and not `library/sysrc` (since it's a file Ansible will copy to the server).